app.json

```diff
@@ -1,24 +1,20 @@
 {
   "name": "work-you-could-do",
+  "repository": "https://github.com/lm-tools/work-you-could-do",
   "scripts": {
     "postdeploy": "bin/rake db:setup"
   },
   "env": {
-    "EMAIL_SENDER"             : { "required" : true  },
-    "GOOGLE_TAG_MANAGER_ID"    : { "required" : true  },
-    "LANG"                     : { "required" : true  },
-    "RACK_ENV"                 : { "required" : true  },
-    "RAILS_ENV"                : { "required" : true  },
-    "RAILS_SERVE_STATIC_FILES" : { "required" : true  },
-    "SECRET_KEY_BASE"          : { "required" : true  }
+    "EMAIL_SENDER":             { "required": true },
```

Comment:
Is this required based on the bits removed on the other commit ?  Or have i looked at the requests in the wrong order and you've already cleaned this out.?